cloud-build/post-install-install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Setup stuff after other installs
+# ? user
+# apt deps:
+
+a2enmod headers
+a2enmod proxy_http
+a2enmod rewrite
+a2enmod ssl
+a2ensite encoded.conf
+a2dissite 000-default
+a2enconf logging
+a2disconf charset
+a2disconf security
+a2disconf localized-error-pages
+a2disconf other-vhosts-access-log
+a2disconf serve-cgi-bin
+if test "%(ROLE)s" = "demo"
```

Comment:
I wonder, given our experience with ENCD-4613 if the batchupgrade should be broken out.  I am 99% sure the order doesn't matter, just that it's supposed to run before the machine reboots with these apache config changes.

Reply:
Yea, that's a good idea.  If it's easy I'll through that in.